core/src/main/java/smashdudes/ecs/systems/CountdownCameraSystem.java

```diff
@@ -0,0 +1,79 @@
+package smashdudes.ecs.systems;
+
+import com.badlogic.gdx.graphics.Camera;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.Vector3;
+import com.badlogic.gdx.utils.Array;
+import smashdudes.ecs.Engine;
+import smashdudes.ecs.Entity;
+import smashdudes.ecs.components.*;
+
+import java.lang.reflect.GenericArrayType;
+
+public class CountdownCameraSystem extends GameSystem
+{
+    private final Vector2 position = new Vector2(0, 0);
+    private int numPlayers;
+    private int currPlayer;
```

Comment:
Maybe store a reference to the playerHandle instead. That way we can't accidentally manipulate it

Reply:
I'm not really sure what you mean